templates/auth.conf.erb

```diff
@@ -84,11 +84,13 @@
 path /puppet/v3/environments
 method find
 allow *
+<% if @puppetversion.to_f < 5.0 -%>
```

Comment:
Hmm , does that need addressing in the proxy then? https://github.com/theforeman/smart-proxy/blob/develop/modules/puppet_proxy_puppet_api/v3_api_request.rb#L14